src/data.ts

```diff
@@ -135,6 +135,8 @@ export interface IBaseOptions {
    * @default 7
    */
   quantiles?: QuantileMethod;
+
+  whiskersMode?: 'nearest' | 'exact';
```

Comment:
can you add a comment similar to the previous ones what this option does?

Reply:
Sure! Done.